scenarios/action_recognition/00_webcam.ipynb

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
<ul><li>This and below looks great!</li><li>Does the generated video look roughly like the Jun Ki's in terms of accuracy?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
should be